support-frontend/app/services/pricing/DefaultPromotionsService.scala

```diff
@@ -0,0 +1,68 @@
+package services.pricing
+
+import com.amazonaws.services.s3.AmazonS3URI
+import com.gu.aws.AwsS3Client
+import com.gu.support.config.{Stage, TouchPointEnvironments}
+import io.circe._
+import io.circe.parser._
+import io.circe.generic.auto._
+import akka.actor.ActorSystem
+import com.gu.support.catalog.{GuardianWeekly, Product}
+import com.typesafe.scalalogging.LazyLogging
+import services.pricing.DefaultPromotionsService.DefaultPromotions
+
+import java.util.concurrent.atomic.AtomicReference
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
+import scala.util.{Failure, Success, Try}
+
+trait DefaultPromotionsService {
+  def getPromoCodes(product: Product): List[String]
+}
+
+object DefaultPromotionsService {
+  case class DefaultPromotions(guardianWeekly: List[String])
```

Comment:
Is the plan that this object will be changed to add additional products going forward? Would it be worth adding them all at the same time?

Reply:
Yes, more products can be added here later. I think it's best to only define the product that's currently supported/required.
Adding another product will involve some more backend changes to the relevant controllers and testing (which I don't want to do now)